assets/src/scripts/interactive/pages/purpose.jsx

```diff
@@ -54,24 +54,26 @@ function Purpose() {
           <Textarea
             characterCount
             className="mb-6"
-            hintText="We should provide some text here to explain"
+            hintText="This title will be added to the top of your generated report. You will be able to change it later."
```

Comment:
How about this to provide more detail? It definitely needs some more work though.

> The title will be shown at the top of the generated report. It should provide a short summary to anyone reading the report about what the analysis shows. For example: "How the COVID-19 pandemic has affected the number of rescue packs being prescribed for COPD patients".
> You will be able to change this after the report is generated.